docs/js/Example.js

```diff
@@ -67,6 +67,17 @@ function getReadableValueType(type) {
   return type.name;
 }
 
+function getDefaultValue(defaultValue) {
+  if (!defaultValue) return '';
+
+  // If default value is a function, show only its name.
+  if (defaultValue.value.indexOf('function') === 0) {
```

Comment:
is this a js fn or a prom fn?

Reply:
and we can use includes! or is it contains....

Reply:
JS function :) I wanted to make the check a bit stricter and make sure the value begins with "function" instead of containing it at a random place :P